tests/e2e/scripts/sync-playwright-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#
+# ---------------------------------------------------------------------
+#
+# GLPI - Gestionnaire Libre de Parc Informatique
+#
+# http://glpi-project.org
+#
+# @copyright 2015-2025 Teclib' and contributors.
+# @licence   https://www.gnu.org/licenses/gpl-3.0.html
+#
+# ---------------------------------------------------------------------
+#
+# LICENSE
+#
+# This file is part of GLPI.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# ---------------------------------------------------------------------
+#
+
+set -e -u -o pipefail
+
+# Path to Makefile
+MAKEFILE="Makefile"
+
+# Extract the version of @playwright/test from package.json
+PLAYWRIGHT_VERSION=$(grep '"@playwright/test"' package.json | sed -E 's/.*"@playwright\/test": *"[^0-9]*([0-9.]+)".*/\1/')
```

Comment:
There is a more robust way to do it.

```suggestion
PLAYWRIGHT_VERSION=$(npm ls @playwright/test --json | jq '.dependencies."@playwright/test".version' | sed 's/"//g')
```